packages/web-app-files/src/index.ts

```diff
@@ -33,60 +34,70 @@ const appInfo = {
   extensions: [],
   fileSideBars
 }
-const navItems = [
-  {
-    name(capabilities) {
-      return capabilities.spaces?.enabled ? $gettext('Personal') : $gettext('All files')
-    },
-    icon: appInfo.icon,
-    route: {
-      path: `/${appInfo.id}/spaces/personal`
-    }
-  },
-  {
-    name: $gettext('Favorites'),
-    icon: 'star',
-    route: {
-      path: `/${appInfo.id}/favorites`
+const navItems = (context) => {
+  return [
+    {
+      name(capabilities) {
+        return capabilities.spaces?.enabled ? $gettext('Personal') : $gettext('All files')
+      },
+      icon: appInfo.icon,
+      route: {
+        path: `/${appInfo.id}/spaces/personal`
+      },
+      enabled(capabilities) {
+        if (!capabilities.spaces?.enabled) {
```

Comment:
the `spaces.enabled` capability check is not needed. For environments without spaces support we create a virtual personal space item and insert it into the runtime vuex store module.
In other words: for environments without spaces support there is **always** a "fake personal space". The check for the absence of a personal space (a few lines below) is already sufficient.